src/PKSim.Infrastructure/ORM/Core/TemplateDatabase.cs

```diff
@@ -8,8 +6,5 @@ public interface ITemplateDatabase : IDatabase
 
    public class TemplateDatabase : AccessDatabase, ITemplateDatabase
    {
-      public TemplateDatabase(): base(CoreConstants.TemplatePassword)
```

Comment:
null and empty is not the same. This was fix in 7.0 but I guess we did not migrate that